parlai/agents/hugging_face/dict.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from parlai.core.dict import DictionaryAgent
+
+try:
+    from transformers import GPT2Tokenizer
```

Comment:
is this right? I know they have their new `tokenizers` repo

Reply:
hmm... i think so. unless the docs are outdated. this is what the generation example in their repo imports https://github.com/huggingface/transformers/blob/ea56d305be0000b3f54ea378df77f92867779ba5/examples/run_generation.py#L31

Reply:
Okay, put it off for later